### PR TITLE
Several toxic fixes - making build green again.

### PR DESCRIPTION
**Pull request by bartoszmajsak**
- Toxiproxy test (`ToxicFunctionalTestCase#shouldAddLatencyWithExec`) using just `.exec()` were causing other tests to fail as it wasn't removing created proxy (this is only performed as part of `then()` block whereas this test simply calls `execute()` instead
- Toxiproxy latest image derives from `alpine`, so we cannot use `ss` as await strategy. This has been changed to `polling`
- Pumba, when using just an image name, brings latest version of the container which does not even start. I locked it to `36-master` which is the latest working image on my machine

### CODE REVIEW

**Comment by bartoszmajsak** [comment]
Problem with using just image names without explicit tags can also hurt users of the library, as it kicked me when just running the build. We should address this problem.

File: ftest-pumba/src/test/resources/arquillian.xml [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
     <extension qualifier="docker">
         <property name="machineName">dev</property>
+      <property name="definitionFormat">CUBE</property>
```
Comment (bartoszmajsak) [inline ftest-pumba/src/test/resources/arquillian.xml]: Not sure if that's needed

File: ftest-toxic-frontend/src/test/resources/arquillian.xml [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
   <extension qualifier="docker">
     <property name="machineName">dev</property>
+    <property name="definitionFormat">CUBE</property>
```
Comment (bartoszmajsak) [inline ftest-toxic-frontend/src/test/resources/arquillian.xml]: Not sure if that's needed

**Comment by bartoszmajsak** [comment]
- [x] Add proxy reset operation to `AfterTest`

**Comment by lordofthejars** [comment]
Cannot see this class `ToxiProxyAfterTestCleaner` in last commit

**Comment by bartoszmajsak** [comment]
Fixes #24

**Pull request merged**